fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -2808,6 +2830,12 @@ bool ClientInfo::canReplace(Reference<ClientInfo> other) const {
 	return !protocolVersion.isCompatible(other->protocolVersion);
 }
 
+std::string ClientInfo::getTraceFileIdentifier(const std::string& baseIdentifier) {
+	std::string versionStr = releaseVersion;
+	std::replace(versionStr.begin(), versionStr.end(), '.', '_');
+	return format("%s_v%st%d", baseIdentifier.c_str(), versionStr.c_str(), threadIndex);
```

Comment:
It's theoretically possible for two external libs to have the same release version, but I would think that is a misconfiguration in any case other than possibly some special dev/testing situations. At any rate, the outcome is not terrible, it just means you would need to revert to looking at the first few traces to see which particular client the file belonged to.

Reply:
In the current design, there can be only one client per protocol version. So it can happen if we have clients with the same package version, but different protocol version. And yes, in the worst case the traces of these clients will share the base name.